openmc/statepoint.py

```diff
@@ -591,17 +644,25 @@ def get_tally(self, scores=[], filters=[], nuclides=[],
             if id and id != test_tally.id:
                 continue
 
-            # Determine if Tally has queried estimator
```

Comment:
This is a bit out of scope, but why iterate over `self.tallies.values()`? This seems like its a dictionary, and doing it this makes it an O(N^2) operation to find all tallies.